db2fce--0.0.11--0.0.12.sql

```diff
@@ -0,0 +1,22 @@
+-- UPPER(text, text): DB2 allows the user to specify a language for the UPPER function.
+-- This is not really needed with PostgreSQL, as upper() uses Unicode, independent of the language.
+-- On DB2, upper('Müller', 'en_US') also returns MÜLLER, just as with de_DE.
+CREATE FUNCTION db2.upper(TEXT, TEXT) RETURNS TEXT AS 
```

Comment:
This change is in both  db2fce--0.0.11--0.0.12.sql and  db2fce--0.0.12.sql → db2fce--0.0.13.sql  and I think it should only be in the latter - if you'd update from 0.0.11 to 0.0.13, you'd run the ```CREATE FUNCTION db2.upper()``` twice resulting in a error, unless I am missing something?